src/libraries/Hooks.sol

```diff
@@ -26,6 +26,7 @@ library Hooks {
     uint256 internal constant AFTER_SWAP_FLAG = 1 << 152;
     uint256 internal constant BEFORE_DONATE_FLAG = 1 << 151;
     uint256 internal constant AFTER_DONATE_FLAG = 1 << 150;
+    uint256 internal constant MODIFY_DELTA_FLAG = 1 << 149;
```

Comment:
thought we were doing a flag for each of the callsites that are allowed to be modified? or is this a TODO still?